added missing icons for newly added fan modes: silent, turbo

ofirzh opened the pull request:
- added the icons.json to provide icons for new fan modes: silent and turbo.
- fan modes names capitalized to have consistent UI

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Added dynamic icon support for the Tornado climate entity, displaying different icons based on fan mode.
  * Introduced English, Hebrew, and Russian translations for the Tornado Air Conditioner integration, including configuration steps, error messages, and climate entity attributes.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->
![Screenshot_20250712_213446_Home Assistant](https://github.com/user-attachments/assets/3d5f3df6-226d-44a3-a055-b3adb8db8f93)
![Screenshot_20250712_213431_Home Assistant](https://github.com/user-attachments/assets/327f3753-a208-48a7-b929-4eda13f2c1a0)

ofirzh commented:
Hey ,
Thanks, It took me whole day. But i'm not 100% sure that it was good to
capitalize fan modes in the code. Without capitalisation in UI the usual
fan modes start with capital letters and silent and turbo lowercased. It
feels like need to research how to do conversion corretly.

On Sat, Jul 12, 2025, 21:28 Rom Freiman ***@***.***> wrote:

> *romfreiman* left a comment
> (romfreiman/tornado-aircon-custom-component#21)
> <https://github.com/romfreiman/tornado-aircon-custom-component/pull/21#issuecomment-3065937859>
>
> Thanks @ofirzh <https://github.com/ofirzh>, I could not find a way to do
> it.
> Mind attaching screenshot to the PR?
>
> Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/romfreiman/tornado-aircon-custom-component/pull/21#issuecomment-3065937859>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA5UWJYOSPCFSSPZRKHDPSL3IFHUDAVCNFSM6AAAAACBL2KJKSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTANRVHEZTOOBVHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

romfreiman commented:
I agree. This is where I stopped as well. I spent some time, but then was not sure what's the right approach and decided to postpone it for later :)

ofirzh commented:
I will check capitalisation and translations in the next weekend.

ofirzh commented:
@romfreiman , looks like I found the right way to do translations.
I reverted the fan modes to lowercase in the code.
I added english translation that does the capitalization and I tested it on russian also.
One screenshot before translation and other after.
<img width="665" height="967" alt="Screenshot 2025-07-13 at 12 33 46" src="https://github.com/user-attachments/assets/ab4c7143-fe42-4f7b-ac55-5007eca25856" />
<img width="629" height="938" alt="Screenshot 2025-07-13 at 12 26 36" src="https://github.com/user-attachments/assets/ab62ee1c-4c65-4ca5-9019-255b7d9a8f38" />

ofirzh commented on custom_components/tornado/icons.json:
```diff
@@ -0,0 +1,31 @@
+            "state": {
+              "auto": "mdi:fan-auto",
+              "low": "mdi:speedometer-slow",
+              "medium": "mdi:speedometer-medium",
```
I tied quickly and  all other attributes including wing direction got default attribute icon if i remember correctly. I can check again tomorrow.

romfreiman commented:
@ofirzh the build failed as well

ofirzh commented:
I saw, but didnt have time to understand what happened there.

ofirzh commented:
@romfreiman, it seems I've found how to do it correctly with minimal changes. Please check on your side also.

ofirzh commented:
> @ofirzh the build failed as well

how to trigger it again?
